src/sif/bicyclecost.cc

```diff
@@ -447,11 +447,11 @@ BicycleCost::BicycleCost(const Costing& costing)
 
   // Get the bicycle type - enter as string and convert to enum
   const std::string& bicycle_type = costing_options.transport_type();
```

Comment:
lets just simply do:
```suggestion
  const std::string& bicycle_type = std::tolower(costing_options.transport_type());
```

and then below we dont need all those `||`s